docs/release_notes.rst

```diff
@@ -1 +1 @@
-.. release-notes:: Release Notes
+.. release-notes:: Release Notes
```

Comment:
What is this change?

Reply:
Lint was giving me some error but then it got fixed. Thanks, I will restore it

Reply:
Updated